prusti-viper/src/encoder/array_encoder.rs

```diff
@@ -19,9 +19,6 @@ use prusti_common::{
     vir_local,
 };
 
-const LOOKUP_PURE_NAME: &str = "lookup_pure";
```

Comment:
Are these stale changes or are we abandoning the approach with common names for `lookup_pure` etc for some reason?